### ISSUE TITLE
Calling functions externally above version 2.2

### ISSUE THREAD

**Issue by ffffffffffffhhhhhhhhhhhh** [sam-k0/GML-Modding-SDK#1]
Can you post an example of calling these functions, I have the name of the functions and I know how to inject a DLL but I do not know how to call it properly!

**Comment by sam-k0** [comment]
Hey there, thanks for your question.
This mod SDK has to be included into the game by the developer, as stated in the description. Think of it like an interface, so game developers don't have to worry about making their game moddable (That has always been a janky thing in GameMaker).
As for version 2.2, I haven't tested it yet.

The general approach would be to include the SDK in your game (You cannot mod games this way if this SDK wasn't imported into the game originally) and then as a modder, write your DLL with this Repo as a base project. 
You can then inject the DLL with any 32bit Injector.

TL;DR you can only mod games with this if the original developer included the extension, if they didn't, you'll have to use tools like Undertale Mod Tool.

Once I have the time I will make a video tutorial on how to use this and share it on the Repo page.
